README.md

```diff
@@ -40,7 +40,8 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 The high-level components of the spec include:
 
 * An archival format for container images, consisting of an [image manifest](manifest.md), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
-* A process of hashing the image format for integrity and content-addressing (base OCI layer)
+* A [process of hashing the image format](descriptor.md) for integrity and content-addressing (base OCI layer)
```

Comment:
It's confusing to link to the top of that document because the first half of it says nothing about the "process of hashing the image format". Probably better to either link to [this section](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests-and-verification) or elsewhere.